config.js

```diff
@@ -0,0 +1,7 @@
+require("dotenv/config");
+
+const config = {};
+
+config.env = process.env.NODE_ENV || "production";
```

Comment:
Why is this not inside the config object?

Reply:
will move